crates/release_plz_core/src/release.rs

```diff
@@ -33,6 +33,8 @@ pub struct ReleaseRequest {
     pub git_release: Option<GitRelease>,
     /// GitHub repo URL.
     pub repo_url: Option<String>,
+    /// Extra flags for `cargo publish`.
+    pub publish_flags: Option<String>,
```

Comment:
Here in the library (release_plz_core), I would prefer to have a Vec<String> instead.

Reply:
Fixed!